### ISSUE TITLE
feature: add a custom `geode://` uri for installing mods from the website

### ISSUE THREAD

**Issue by pauliesnug** [geode-sdk/website#48]
it would launch geode automatically and download the mod from the uri. the uri would have the mod index id similar to how celeste's olympus lets you install using a custom uri.

**Comment by Fleeym** [comment]
I think the idea has been thrown around in the past. First idea would be to do it using the CLI, but most users won't do that

**Comment by HJfod** [comment]
We could bundle some sort of installation handling app alongside Geode that justs deals with this. Or we could have a launcher for Geode that handles this alongside managing mods before starting up the game etc.

**Comment by pauliesnug** [comment]
> We could bundle some sort of installation handling app alongside Geode that justs deals with this. Or we could have a launcher for Geode that handles this alongside managing mods before starting up the game etc.

I think a Geode launcher would work really well. Right now we have the packages. I think we could take inspiration in how EverestAPI does the mod downloading and profile management all in one centralized launcher @HJfod. I'd love to spearhead a launcher if that helps.

**Comment by coopeeo** [comment]
oh my god i was thinking of doing this too, (as well as a sharing mod,) but I have no clue how I would handle it in a mod.

**Comment by pauliesnug** [comment]
> oh my god i was thinking of doing this too, (as well as a sharing mod,) but I have no clue how I would handle it in a mod.

i think it would be better if it was a launcher io
